microprofile.jdt/com.redhat.microprofile.jdt.core/src/main/java/io/quarkus/runtime/util/StringUtil.java

```diff
@@ -0,0 +1,220 @@
+package io.quarkus.runtime.util;

```

Comment:
please add link to the original source, and document the original license

Reply:
I add links to the original Quarkus StringUtil class.

Reply:
License Apache 2 has been added.